tools/kochdocs.nim

```diff
@@ -14,10 +14,10 @@ const
   webUploadOutput = "web/upload"
 
 var nimExe*: string
-const allowList = ["jsbigints.nim"] # Nim for JavaScript target outside of lib/js/
```

Comment:
sync bug? how can that work without this

Reply:
https://github.com/nim-lang/Nim/pull/16644#issue-551902996 seems flaky already.

Reply:
what do you mean? `const allowList = ["jsbigints.nim"] ` is unrelated to https://github.com/nim-lang/Nim/pull/16644; I'm not sure why you removed `const allowList = ["jsbigints.nim"] ` after rebasing or re-syncing